js.js

```
@@ -0,0 +1,52 @@
+var num1 = 10,num2 = 20;
```

Biome

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)